CookSmart/CookSmart/IngredientListViewController.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  IngredientListViewController.swift
+//  cake
+//
+//  Created by Alex King on 3/28/20.
+//  Copyright © 2020 Olga Galchenko. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+class IngredientListViewController: UIHostingController<IngredientListView> {
+  init() {
+    super.init(rootView: IngredientListView())
+  }
+
+  @objc
+  dynamic required init?(coder _: NSCoder) {
+    assertionFailure("init(coder:) has not been implemented")
+    return nil
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    CSIngredients.sharedInstance()?.refreshRecents()
```

Comment:
Hmmm, why is the CSIngredients optional? When might it be missing? Might it make sense to just throw when it's not available inside the `sharedInstance` implementation and leave the return type non-optional.

Reply:
or is the optional just the artifact of Objective C interop?